sonar-db/src/test/java/org/sonar/db/permission/GroupWithPermissionDaoTest.java

```diff
@@ -54,111 +54,12 @@
 
   @Rule
   public DbTester db = DbTester.create(System2.INSTANCE);
-  GroupDbTester groupDb = new GroupDbTester(db);
-  PermissionDbTester permissionDb = new PermissionDbTester(db);
-  ComponentDbTester componentDb = new ComponentDbTester(db);
-  DbSession dbSession = db.getSession();
+  private GroupDbTester groupDb = new GroupDbTester(db);
```

Comment:
remark: in test I don't think it's necessary to have private fields, it make the test a little bit more readable IMO